e2e/collections_test/test_contains5/test.yml

```diff
@@ -0,0 +1,9 @@
+
+query: >
+    MATCH p = ()-[*]->()
+    WITH collect(p) AS paths
+    MATCH q = (:Dog)-[*]->(:Human)
+    CALL collections.contains(paths, q) YIELD output RETURN output;
+output:
+  - output: true
+
```

Comment:
This shouldn't also be like this, you can check input.cyp of this test, this is how it is supposed to be :smile: